forge-gui/res/cardsfolder/upcoming/tura_kennerud_skyknight.txt

```diff
@@ -0,0 +1,10 @@
+Name:Tura Kennerüd, Skyknight
```

Comment:
I think this `ü` is what's causing your tests to fail

I know we don't use `û` for stuff like Lim-Dhûl so maybe just remove the umlaut?